scripts/createShadowCpp/createShadow.cpp

```diff
@@ -334,7 +384,7 @@ int main(int argc, const char** argv) {
     // if (compact) {
     //     useCompact = true;
     // }
-    GraphType g = GraphType::gw;
+    //GraphType g = GraphType::gw;
```

Comment:
We are not using GraphType anymore?

Reply:
It was an unused variable warning during compile time, so I guess not?